BrowserKit/Sources/ToolbarKit/LocationView.swift

```diff
@@ -47,7 +53,9 @@ class LocationView: UIView, UITextFieldDelegate, ThemeApplicable {
         static let horizontalSpace: CGFloat = 8
         static let gradientViewVerticalPadding: CGFloat = 8
         static let gradientViewWidth: CGFloat = 40
-        static let clearButtonSize: CGFloat = 40
+        static let urlTextFieldViewsSize: CGFloat = 40
+        static let searchEngineImageViewSize: CGFloat = 24
+        static let cornerRadius: CGFloat = 4
```

Comment:
Let's rename this to make it clearer for where the cornerRadius is used: `searchEngineImageCornerRadius`